Vault Page Updates (#4677)
w3f/polkadot-wiki: docs/general/polkadot-vault.md

|              Hardware designed to stay offline and be secure. Certified by French cybersecurity agency [ANSSI](https://www.ssi.gouv.fr/en/cybersecurity-in-france/the-national-cybersecurity-agency-of-france/).               |                                                                                                                                                                                                                         Hardware not designed to stay offline. The user must switch off all inbound and outbound connections (network, wifi, Bluetooth, NFC).                                                                                                                                                                                                                          |

> |         Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/). The secure element will detroy itself if opened.         |                                                                                                                                                                                                    Although there is no wired connection, side-channel attacks are possible. Without a secure element, the phone can be opened and the keys will be accessible in its storage unit.                                                                                                                                                                                                    |

Reviewer: ```suggestion
|         Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/). The secure element will destroy itself if opened.         |                                                                                                                                                                                                    Although there is no wired connection, side-channel attacks are possible. Without a secure element, the phone can be opened, and the keys will be accessible in its storage unit.                                                                                                                                                                                                    |
```

| Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                                                                                                  Metadata updates are always available to install once released, either through the third-party provider or manually. In this case, the installation process requires familiarity with working on the command prompt.                                                                                                                                                                                  |

## Create and Import Accounts

You can create a new account directly within the Vault app (Add Key Set > Add new Key Set). This
will generate a new mnemonic seed phrase on the app. Alternatively, you can import a new account in

> the app using a seed phrase generated on a cold device elsewhere. For example, you can have an

Reviewer: Can be a bit misleading. For instance, can we import the account created on Ledger?

```suggestion
the app using a seed phrase generated elsewhere with a compatible account generation scheme. For example, you can have an
```

[this article](https://support.polkadot.network/support/solutions/articles/65000184118-polkadot-vault-how-to-add-your-account-on-polkadot-js-ui)
on how to import them to the Polkadot extension or Polkadot-JS UI.

:::info

Importing an account into an extension will not import its private key. Only the public key will be

> imported and you must sign using your air-capped phone (as the private key is there).

Reviewer: ```suggestion
imported and you must sign using your air-gapped phone which holds the private key.
```
